lib/components/viewers/styled.js

```diff
@@ -50,19 +55,21 @@ export const StopContainer = styled.div`
   are shown when browsers don't calculate 100% sensibly */
   padding: 15px 0 100px;
 `
-export const Stop = styled.a`
-  cursor: pointer;
+export const StopLink = styled.a`
   color: ${(props) => props?.textColor + 'da' || '#333'};
+
+  &:hover {
+    color: ${(props) => props?.textColor || '#23527c'};
```

Comment:
What do you think about bringing back the slight transparency? I thought it looked nice

Reply:
The transparency is still on the link on line 59, isn't it? I think it's just gone on `:hover`! Agree it looks nice!!